std_msgs/Float32MultiArray rotation # 3x3 rotation matrix in row major order
std_msgs/Float32MultiArray translation # xyz
std_msgs/Header header
